include/vinecopulib/vinecop/class.hpp

```diff
@@ -81,12 +81,16 @@ class Vinecop
   void to_json(const std::string filename) const;
 
   // Methods modifying structure and/or families and parameters
-  void select_all(Eigen::MatrixXd data,
-                  const FitControlsVinecop& controls = FitControlsVinecop());
+  void select(Eigen::MatrixXd data,
+              const FitControlsVinecop& controls = FitControlsVinecop());
+
+  void DEPRECATED
+  select_all(Eigen::MatrixXd data,
+             const FitControlsVinecop& controls = FitControlsVinecop());
 
-  void select_families(
-    Eigen::MatrixXd data,
-    const FitControlsVinecop& controls = FitControlsVinecop());
+  void DEPRECATED
```

Comment:
we use `DEPRECATED void` in `FitControlsVinecop` any reason for doing it the other way around?